tests/reference/run_dbg-test_quit_01-30889cc.stderr

```diff
@@ -0,0 +1,9 @@
+  File "tests/runtime_errors/test_quit_01.py", line 4
+    test()
+  File "tests/runtime_errors/test_quit_01.py", line 1
+    def test():
+  File "tests/runtime_errors/test_quit_01.py", line 4
+    test()
+  File "tests/runtime_errors/test_quit_01.py", line 2
+    quit(10)
+STOP
```

Comment:
These tests would fail in MacOS, I think we have to implement runtime_stacktrace in MacOS as well.

Reply:
I need to implement a macOS support for this.